irohad/main/application.hpp

```diff
@@ -100,8 +103,7 @@ class Irohad {
    * (optional). If not provided, disables mst processing support
    * TODO mboldyrev 03.11.2018 IR-1844 Refactor the constructor.
    */
-  Irohad(const std::string &block_store_dir,
```

Comment:
Not sure if it will be possible to smoothly integrate it by removing the FlatFile directly now, because users will probably want to migrate their storage from files to postgres. Need input from @MBoldyrev here

Reply:
I do not think we need a universal migration solution coded into irohad. IMHO we should better provide a reference python script for adaption to particular migration cases when someone needs it.